torch/csrc/jit/codegen/cuda/transform_rfactor.cpp

```diff
@@ -198,7 +198,7 @@ TensorDomain* TransformRFactor::runReplay(
     for (auto id : orig_td->domain()) {
       if (axes_set.find(i++) != axes_set.end())
         rfactor_axes.emplace(id);
-      if (id->isReduction())
+      else if (id->isReduction())
```

Comment:
Good catch! Let's also fix the coding style so that it doesn't omit `{` and `}`.